README.md

```diff
@@ -0,0 +1,40 @@
+# lsst-rsp
+
+User-facing Python classes and functions for use in the RSP environment.
+Learn more at https://lsst-rsp.lsst.io
+
+Install from PyPI:
+
+```sh
+pip install lsst-rsp
+```
+
+lsst-rsp is developed by Rubin Observatory at https://github.com/lsst-sqre/lsst-rsp.
+
+## Features
+
+<!-- A bullet list with things that this package does -->
+
+## Developing lsst-rsp
+
+The best way to start contributing to lsst-rsp is by cloning this repository, creating a virtual environment, and running the `make init` command:
+
+```sh
+git clone https://github.com/lsst-sqre/lsst-rsp.git
+cd lsst-rsp
+make init
+```
```

Comment:
It'd be nice to have instructions on how to test this inside of a jupyter container since that's the only real place where it's useful

Reply:
Also done.